client/state/comments/reducer.js

```diff
@@ -111,7 +111,11 @@ export function items( state = {}, action ) {
 				[ stateKey ]: map( state[ stateKey ], updateComment( commentId, comment ) ),
 			};
 		case COMMENTS_RECEIVE:
-			const { skipSort, comments } = action;
+			const { skipSort } = action;
+			const comments = map( action.comments, cmt => ( {
```

Comment:
`comment` over `cmt`?

Reply:
Hmm.  I think i remember there being a name conflict. I'll fix

Reply:
yep. theres a name conflict.  I'll make it `_comment`